test_benchmarks.py

```
@@ -41,3 +44,84 @@ def test_save_chunks_benchmark(benchmark, tmp_path):
+            url = "http://example.com/benchmark"
+            scraper.scrape_website(url)
+            
+            # Benchmark subsequent requests
+            def scrape():
+                return scraper.scrape_website(url)
+            
+            benchmark(scrape)
+
+
+@pytest.mark.benchmark(group="caching")
+def test_benchmark_scrape_with_cache_disabled(benchmark):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_benchmark_scrape_with_cache_disabled`

Add return type annotation: `None`

(ANN201)